### DISCUSSION TITLE
how i can add vuejs-smart-table library to vitesse ssg

### DISCUSSION THREAD

**Discussion by i-s-a2021** [antfu/vitesse#277] [Q&A] [answered]
hi
I'm not very familiar with "vitesse-ssg starter template"
I want to add a new library "vuejs-smart-table" to use it but i can't use App.use(SmartTable) in App.vue or in main.ts ...
what I should use in "vitesse ssg" instead of using App.use(SmartTable)

import SmartTable from 'vuejs-smart-table'

Vue.use(SmartTable)

**Comment by userquin** [comment, marked as answer]
Check this  https://github.com/antfu/vite-ssg/issues/129#issue-1053911731

**Reply by i-s-a2021** [reply]
thank you

**Comment by thuongtruong1009** [comment]
Hi @i-s-a2021 !
- You can use the external library by importing and declaring them in the auto-imports.d.ts file.
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this.
- Thank you!

**Reply by i-s-a2021** [reply]
thank you
